### PR TITLE
Checks were failing due to lack of quotes

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: dgalloway <dgallowa@REDHAT.COM>

### CODE REVIEW

File: roles/common/tasks/rhel-entitlements.yml [hunk 1]
```diff
@@ -69,14 +69,14 @@
 
 - name: Disable all rhsm repos
   command: subscription-manager repos --disable '*'
-  when: replace_repos == True
+  when: replace_repos == "True"
```
Comment (andrewschoen) [inline roles/common/tasks/rhel-entitlements.yml]: I think I'd prefer is this was just ``when: replace_repos``.

File: roles/common/tasks/rhel-entitlements.yml [hunk 2]
```diff
@@ -69,14 +69,14 @@
...
   # This produces an absurd amount of useless output
   no_log: True
 
 - name: Enable necessary rhsm repos
   command: subscription-manager repos --enable {{ item }}
   with_items: rhsm_repos
-  when: replace_repos == True and rhsm_repos|length > 0
+  when: replace_repos == "True" and rhsm_repos|length > 0
```
Comment (andrewschoen) [inline roles/common/tasks/rhel-entitlements.yml]: Same comment as above.

Comment (dmick) [inline roles/common/tasks/rhel-entitlements.yml]: Is string truth really right?   Is there no Boolean?  Also, is "a" true?

Comment (andrewschoen) [inline roles/common/tasks/rhel-entitlements.yml]: We explicitly set ``replace_repos`` to either True or False.  From my understand (true, True, False and false) will all be made a boolean by jinja2.  If we wanted we could do an explicit cast to a bool with: ``when: "{{ replace_repos|bool }}"`` I believe.

See: http://docs.ansible.com/ansible/playbooks_conditionals.html#the-when-statement

**Pull request closed without merging**